src/test/java/com/klarna/hiverunner/PreV130HiveShellHiveCliEmulationTest.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (C) 2013-2018 Klarna AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.klarna.hiverunner;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.klarna.hiverunner.annotations.HiveRunnerSetup;
+import com.klarna.hiverunner.annotations.HiveSQL;
+import com.klarna.hiverunner.config.HiveRunnerConfig;
+import com.klarna.hiverunner.sql.cli.hive.PreV130HiveCliEmulator;
+
+@RunWith(StandaloneHiveRunner.class)
+public class PreV130HiveShellHiveCliEmulationTest {
+
+  @HiveRunnerSetup
+  public final static HiveRunnerConfig CONFIG = new HiveRunnerConfig() {{
+      setCommandShellEmulation(PreV130HiveCliEmulator.INSTANCE);
+  }};
+  
+  @HiveSQL(files = {}, encoding = "UTF-8")
```

Comment:
Uncertain, could you use com.google.common.base.Charsets.UTF_8 here instead of the String?

Reply:
prefer jdk over guava: java.nio.charset.StandardCharsets.UTF_8